packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -8,8 +8,8 @@ class DateTimeUtils {
    * @param {string} iSODate - The ISO string
```

Comment:
There are alot of changes in these utils files. Are there jest tests to go with them?

Reply:
Yeah let's definitely add some tests.

Reply:
Added here 2b9b95282e579abb629029b54bd70a7c3ec0049a

Reply:
@lokesh-0813 the tests you added were wdio tests. I think Jest test here to validate these function would be more reliable.

Reply:
@emilyrohrbough  are you talking about tests from this commit 2b9b95282e579abb629029b54bd70a7c3ec0049a ?

Reply:
Oops! wasn't looking close enough. This is good. 👍